guests/reth-ethereum/Cargo.toml

```diff
@@ -5,6 +5,19 @@ edition = "2021"
 
 [workspace]
 
+[profile.dev]
+opt-level = 3
+
+[profile.dev.build-override]
+opt-level = 3
```

Comment:
`build-override` isn't needed anymore

Reply:
somehow I think this is still affecting cycle count after a clean build

Reply:
As I understand the docs, `build-override` only affects how build scripts etc. are built, not the actual crate, so it should have no effect on the cycle count except for some random noise.

Reply:
In that context, I don't even understand how the dev profile affects cycle counts, since everything is supposed to be built in release.